fastanvil/src/java/pre13/pre13_block_names.rs

```diff
@@ -85,10 +84,13 @@ fn modern_block(block_name: &'static str, data_value: u8) -> Block {
                 1 => "top",
                 _ => unreachable!(),
             };
+
+            let mut properties = HashMap::new();
+            properties.insert("type".to_string(), top.to_string());
+
             Block {
                 name: format!("{kind}_slab"),
```

Comment:
I assume this (and others) need a `minecraft:` in front of them. Is this a bug? Didn't want to change it before asking